### PR TITLE
Master

### PR DESCRIPTION
**Pull request by mikeyb**
Clarity around contribution steps.  Previously just said fork, fix, pr.  But did not include what to branch from and merge to.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -7,12 +7,28 @@ and help.
...
+complex changes though, please check up with the core devs first on [our gitter channel](https://gitter.im/ethereumproject/go-ethereum)
+to ensure those changes are in line with the general philosophy of the project and/or get some
+early feedback which can make both your efforts much lighter as well as our review and merge
+procedures quick and simple.
+
+Please make sure your contributions adhere to our coding guidelines:
+
+ * Code must adhere to the official Go [formatting](https://golang.org/doc/effective_go.html#formatting) guidelines (i.e. uses [gofmt](https://golang.org/cmd/gofmt/)).
+ * Code must be documented adhering to the official Go [commentary](https://golang.org/doc/effective_go.html#commentary) guidelines.
+ * Pull requests need to be based on and opened against the `develop` branch.
```
Comment (splix) [inline .github/CONTRIBUTING.md]: there also

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -7,12 +7,28 @@ and help.
...
+The maintainers will review and merge into the main code base. If you wish to submit more
+complex changes though, please check up with the core devs first on [our gitter channel](https://gitter.im/ethereumproject/go-ethereum)
+to ensure those changes are in line with the general philosophy of the project and/or get some
+early feedback which can make both your efforts much lighter as well as our review and merge
+procedures quick and simple.
+
+Please make sure your contributions adhere to our coding guidelines:
+
+ * Code must adhere to the official Go [formatting](https://golang.org/doc/effective_go.html#formatting) guidelines (i.e. uses [gofmt](https://golang.org/cmd/gofmt/)).
+ * Code must be documented adhering to the official Go [commentary](https://golang.org/doc/effective_go.html#commentary) guidelines.
```
Comment (pascaldekloe) [inline .github/CONTRIBUTING.md]: The suggestions from Effective Go are quite nice indeed which is exceptional for convention guides. Could you also mention the names section?

File: README.md [hunk 3]
```diff
@@ -250,8 +250,13 @@ limit blocks converge to (`--targetgaslimit`) and the price transactions are acc
...
 
-If you'd like to contribute to go-ethereum, please fork, fix, commit and send a pull request
-for the maintainers to review and merge into the main code base. If you wish to submit more
+If you'd like to contribute to go-ethereum:
+
+ *  Fork `go-ethereum` repo
+ *  Branch from `master`
+ *  Commit fixes send a pull request back to `master`
+
+The maintainers will review and merge into the main code base. If you wish to submit more
```
Comment (pascaldekloe) [inline README.md]: This is redundant. Better refer tho CONTRIBUTING.md instead.

**Comment by pascaldekloe** [comment]
No response.

**Comment by mikeyb** [comment]
Lol.  No cares.  You prefer crap documentation :)

**Pull request closed without merging**